### PR TITLE
[2544] Redirect login/register

### PR DESCRIPTION
**Pull request by blazarecki**

### CODE REVIEW

File: app/config/routing.yml [hunk 1]
```diff
@@ -108,6 +108,21 @@ app_redirecting:
     resource: "@AppBundle/Controller/RedirectingController.php"
     type:     "annotation"
 
+
```
Comment (HeahDude) [inline app/config/routing.yml]: Extra empty line

Comment (blazarecki) [reply app/config/routing.yml]: thanks, fixed

File: features/redirect.feature [hunk 2]
```diff
@@ -0,0 +1,18 @@
+@wip
```
Comment (B-Galati) [inline features/redirect.feature]: :trollface:

**Comment by blazarecki** [comment]
@B-Galati fixed

**Pull request merged**